config/rbac/workloads_replicatedstatemachine_editor_role.yaml

```diff
@@ -1,20 +1,20 @@
-# permissions for end users to edit consensussets.
```

Comment:
need to add new roles to helm chart as well

Reply:
done: [5a1592f](https://github.com/apecloud/kubeblocks/pull/3733/commits/5a1592f0568ea898647608c912bddb95fcac189b)